.github/workflows/sync_tags_localise.yml

```diff
@@ -0,0 +1,46 @@
+name: Synchronize Github Tags with Localise tags
+
+on:
+    workflow_call:
+    workflow_dispatch:
+    push:
+        tags:
+            - '*'
```

Comment:
The update_locales.yml is also executed when we push a tag, and it calls this workflow so this will be executed twice. In this case, we should remove those 3 lines so it's only called once